src/fxos/index.js

```diff
@@ -0,0 +1,13 @@
+let modularize = require('../modularize');
+
+let Fxos = modularize(function Fxos(adb) {
```

Comment:
`Fxos` is just business logic on top of `Adb`, so request-specific `Adb` means request specific `Fxos`.